Headers/AppKit/NSSequePerforming.h

```diff
@@ -0,0 +1,47 @@
+/* Definition of class NSSequePerforming
```

Comment:
This class and the file and of course the implementation file as well should be called NSSeguePerforming. ("g" instead of "q")

Reply:
Fixed.